raiden/transfer/state.py

```diff
@@ -1498,10 +1506,8 @@ def from_dict(cls, data: Dict[str, Any]) -> 'NettingChannelEndState':
             data['secrethashes_to_onchain_unlockedlocks'],
         )
         restored.merkletree = data['merkletree']
-
-        balance_proof = data.get('balance_proof')
-        if data is not None:
-            restored.balance_proof = balance_proof
+        restored.balance_proof = data.get('balance_proof')
```

Comment:
So restored balance proof can be None here?

Reply:
Yes, the old code and the new code do the same thing pretty much right? not setting balance proof is implicitly setting it to None